### PR TITLE
Bug 1942915: pkg/.../create_error_template: fix error page template

### PR DESCRIPTION
**Pull request by s-urbaniak**
/cc @stlaz

### CODE REVIEW

File: pkg/cli/admin/createerrortemplate/create_error_template.go [hunk 1]
```diff
@@ -74,8 +74,9 @@ func (o *CreateErrorTemplateOptions) Run() error {
 const ErrorPageTemplateExample = `<!DOCTYPE html>
 <!--
 
-This template can be modified and used to customize the error page. To replace
-the error page, set master configuration option oauthConfig.templates.error to
+This template can be modified and used to customize the error page for oauth identity providers and the request header identity provider.
```
[the remark this answers is not in the document]
Comment (s-urbaniak) [reply pkg/cli/admin/createerrortemplate/create_error_template.go:77]: Indeed, that sounds more clear :+1:

File: pkg/cli/admin/createerrortemplate/create_error_template.go [hunk 2]
```diff
@@ -74,8 +74,9 @@ func (o *CreateErrorTemplateOptions) Run() error {
...
+The template does not have effect on password based identity providers using the login page.
```
[the remark this answers is not in the document]
Comment (s-urbaniak) [reply pkg/cli/admin/createerrortemplate/create_error_template.go:78]: sounds good :+1:

**Comment by s-urbaniak** [comment]
@stlaz fixed wording, ptal

**Review by soltysh — APPROVED** [review]
/approve
/retest

**Pull request merged**